nvflare/app_common/pt/pt_model_reader_writer.py

```diff
@@ -26,14 +24,11 @@ def __init__(self):
         self._name = self.__class__.__name__
         self.logger = logging.getLogger(self._name)
```

Comment:
The name `PTModelReaderWriter` seems to suggest we are reading / writing anything to file or byte stream or anything.

But the code is doing Extract model parameters from a PyTorch network and return a dict.
And Loading the model parameters back into the PyTorch network.

I suggest we change this name to something more accurate.
Some ideas like: `PTModelProcessor` or `PTModelDictConverter`